tests/test_defect.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from bugbug.models.defect import DefectModel
+
+
+def test_get_defect_labels():
+    model = DefectModel()
+    classes, _ = model.get_labels()
+    assert classes[1042414] == 1
+    assert not classes[1049816] == 1
```

Comment:
As in your other PR, `assert not classes[1049816] == 1` is confusing, either `assert not classes[1049816]` or `assert classes[1049816] != 1` are better.

Reply:
Okay, I switched it to use '!='. The other test PR is actually someone else's though, not mine.

Reply:
Ahah, I didn't notice :P